tests/unit/stores/utilities/pool.spec.js

```diff
@@ -28,11 +30,7 @@ describe('pool', () => {
       expect(usedAliquot.volume).toBe(10)
       expect(usedAliquot.concentration).toBe(5)
       expect(usedAliquot.insert_size).toBe(1)
-    })
-    it('correctly sets the default volume when initialiseToAvailableVolume is provided', () => {
-      const initialiseToAvailableVolume = () => '20'
-      const usedAliquot = createUsedAliquot({ volume: 10 }, initialiseToAvailableVolume)
-      expect(usedAliquot.volume).toBe(20)
+      expect(usedAliquot.available_volume).toBe(10)
     })
```

Comment:
Is there a test already exists either in unit or e2e to check the context where we are adding the same library twice in a run, to ensure that this fix is covered? If not, would it be better to add one?

Reply:
Yes there is an e2e test for it